src/holon/migrations/0048_alter_datamodelqueryrule_datamodel_conversion_step.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 4.1.9 on 2023-06-13 07:13
+
+from django.db import migrations
+import django.db.models.deletion
+import modelcluster.fields
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("holon", "0047_alter_actor_payload_alter_rule_model_type"),
```

Comment:
In the meantime new migrations are added in main. So an extra merge migration needs to be added or this migration needs to be the 52th migration and be dependent 51. If not django can't migrate

Reply:
Done